catalog/ui/search-ui/standard/src/main/webapp/js/controllers/cesium.controller.js

```diff
@@ -59,10 +62,23 @@ define(['application',
                 this.listenTo(wreqr.vent, 'metacard:overlay', this.overlayImage);
                 this.listenTo(wreqr.vent, 'metacard:overlay:remove', this.removeOverlay);
 
+                this.clustering = false;
+
                 if (wreqr.reqres.hasHandler('search:results')) {
                     this.newResults(wreqr.reqres.request('search:results'));
                 }
             },
+
+            toggleClustering: function() {
+                if(!this.clustering) {
+                    this.clustering = true;
```

Comment:
I think `MapClustering` should track the clustering state instead of this controlling trying to stay in sync with it.  You could also add a `toggle` method in cases where you do not need to know the previous status but just want to switch it.